asllib/doc/ASLSemanticsReference.tex

```diff
@@ -6401,7 +6265,7 @@ \subsection{Prose}
   (\overname{\Bool}{\vb} \times \overname{\XGraphs}{\vg}) \cup \overname{\TError}{\ErrorConfig}
 \]
 checks whether the value $\vv$ can be stored in a variable of type $\vt$ in the environment $\env$,
-resulting in a Boolean value $\vb$ and execution graph $\vg$, \ProseOrError.
+resulting in a Boolean value $\vb$ and execution graph $\vg$ or a dynamic error.
```

Comment:
why drop the macro here?

Reply:
this macro should only appear in premises.